app/src/androidTest/java/com/pajato/android/gamechat/FABTest.java

```diff
@@ -38,24 +39,54 @@
         // Ensure that the speed dial Indicators are all visible.
         onView(withText(R.string.new_chat_select_user))
                 .check(matches(isDisplayed()));
+        onView(withText(R.string.new_chat_favorite_room))
+                .check(matches(isDisplayed()));
         onView(withText(R.string.new_game_settings))
                 .check(matches(isDisplayed()));
-        onView(withText(R.string.new_chat_favorite_room))
+        onView(withText(R.string.new_game_ttt))
                 .check(matches(isDisplayed()));
     }
 
-    /** Ensure that, when navigating to the game pane, the FAB disappears. *
-     * NOTE: This test is problematic on tablets, and has been commented out until the time comes
-     * that a better test is written to check this functionality.
-    @Test public void testFABNotDisplayedGameFragment() {
-        // Ensure the FAB is present. Then, Navigate to the game fragment.
+    @Test public void testFabFunctionality() {
+        onView(withId(R.id.chat_pane))
+                .check(matches(isDisplayed()));
+        // Open up the FAB menu and click on the new Tic-Tac-Toe game option
```

Comment:
fwiw, I am inclined to have whitespace before a comment, especially where it helps call attention to the comment.